### ISSUE TITLE
Not needed

### ISSUE THREAD

**Issue by awinogradov** [posthtml/posthtml-site#1]
PostHTML will have official landing page at theprotein.io/posthtml ;)

**Comment by awinogradov** [comment]
@michael-ciniawsky can you move this to personal repo?

**Comment by michael-ciniawsky** [comment]
that was not initially created by me if it's on theprotein.io/posthtml and here will be no project here delete it. But i was waiting for maltsev to dicuss how/if the plugins catalog maybe find its place on a future side, thats obselete then

**Comment by awinogradov** [comment]
Ok) plugins catalog can be separated page anywhere and it can have api for everybody.

**Comment by awinogradov** [comment]
@maltsev we need you)

**Comment by michael-ciniawsky** [comment]
The questuon is why not make the website at PostHTML the repo can resign on my/anyones github while developing as any other repo can if this approach is better in general

i planned togo for it in this way:

posthtml.tld
api.posthtml.tld (genrated from PostHTML JSDoc)
plugins.posthtml.tld (plugins and plugins, plugins api to add/search plugins, plugin docs generated form JSDoc)

**Comment by michael-ciniawsky** [comment]
of course approved by all but for the architecture

**Comment by awinogradov** [comment]
Because PostHTML is the product of Protein team. 

Another way:
http://theprotein.io/posthtml - stored in https://github.com/theprotein
http://theprotein.io/posthtml/plugins - can be stored in https://github.com/posthtml
http://theprotein.io/posthtml/docs - API should be combined automatically from https://github.com/posthtml/posthtml, because everybody must be able to read it offline in posthtml module.

It's common arch for our products.

**Comment by michael-ciniawsky** [comment]
yep alright i didnt know about your internals it's no offense that the relation to the protein must be crystal clear i know but not how it could also be redirected with DNS so both works but do as you prefer :)

**Comment by awinogradov** [comment]
We use pointDNS for this. Don't worry;)

**Comment by michael-ciniawsky** [comment]
Could someone please either delete this repo or rename it to posthtml.js.org and use dns.js.org as domain, like e.g webpack.js.org does. It would be also possible to serve it from e.g docs/ folder in posthtml/posthtml, depending on your prefernce :) ?

**Comment by michael-ciniawsky** [comment]
I will make a website soon, going in the direction like https://vuejs.org/v2/guide/ in terms of design
